### DISCUSSION TITLE
How to delete plugin from DSA

### DISCUSSION THREAD

**Discussion by abhisheklomsh** [andreped/FP-dsa-plugin#7] [Q&A] [answered]
As we upload plugin images from "Tasks / Slicer CLI Web Tasks" we are going to get drop-down under "analyses" tab, I want to know how to delete them.

To show the dropdown under "analyses" tab I am attaching a screenshot.

<img width="563" alt="Screenshot 2023-09-06 at 7 45 25 PM" src="https://github.com/andreped/FP-DSA-plugin/assets/13444259/c8df593e-9664-4f0a-96de-423ee4bd3756">

**Comment by andreped** [comment]
Hmm, I have never deleted a plugin from DSA, tbh, but I would think that if you go to DSA, not the HistomicsUI, you should be able to delete the plugin from the `Slicer CLI Web Tasks` in the `Tasks` collection. Same place as where you add new plugins.

Go to [this](https://demo.kitware.com/histomicstk/#collection/60d4c36ff98473b86b6450d5/folder/60d4c36ff98473b86b6450d6) website to see where to delete the plugin.

**Reply by abhisheklomsh** [reply]
Thanks @andreped for the response, still trying to find a solution for this, checked the link however there isnt a delete option, trying to dig into the code to locate it.

**Reply by andreped** [reply]
I saw that you opened a Discussion at the DSA repo. That's probably the place to ask DSA-related questions.

If you find the answer to the problem, be sure to post it here, as it could be of interest to others.

**Reply by andreped** [reply, marked as answer]
I just tried deleting my own plugin, and that worked fine. Same goes for the `histomicstk` plugin.

You need to go inside the plugin folder under `Slicer CLI Web Tasks` and **delete the folder**. That should remove the plugin.

<img width="1677" alt="Screenshot 2023-09-07 at 08 45 30" src="https://github.com/andreped/FP-DSA-plugin/assets/29090665/7df6cb62-7aca-4911-a578-69255d20b657">

Here you can see that it has now been removed:

<img width="944" alt="Screenshot 2023-09-07 at 08 47 54" src="https://github.com/andreped/FP-DSA-plugin/assets/29090665/4c8dceba-f16d-4464-866d-ee3adb1cf658">

**Reply by abhisheklomsh** [reply]
I am able to access these settings under "User" section but not Collections>Tasks side of things
When I hover mouse to folder actions a label comes up which says they are disabled, I am accessing as an admin.

**Reply by abhisheklomsh** [reply]
Got it now, if I tick checkbox of anything be it file or folder, then these folder actions get disabled.

Thanks, resolved

**Reply by andreped** [reply]
As this resolved your issue, it would be great if you could mark my comment as answer.

If you have any future `FP-DSA-plugin`related questions, feel free to open a new Q/A Discussion :]
